src/core/components/Footer/ContactInfo.tsx

```diff
@@ -0,0 +1,29 @@
+import { faBriefcase, faEnvelope, faFile, faMobileAlt } from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import React from 'react';
+import style from './footer.less';
+
+const ContactInfo = () => (
+  <div className={style.contact}>
+    <div>
+      <FontAwesomeIcon icon={faBriefcase} fixedWidth className={style.icon} /> 992 548 045 (OrgNr) <br />
+      <FontAwesomeIcon icon={faEnvelope} fixedWidth className={style.icon} /> kontakt@online.ntu.no <br />
+      <FontAwesomeIcon icon={faFile} fixedWidth className={style.icon} /> okonomi@online.ntnu.no <br />
+      <FontAwesomeIcon icon={faMobileAlt} fixedWidth className={style.icon} /> 73 59 64 89
+    </div>
+    <div>
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/address

Reply:
Will fix, what kind of images? :P

Reply:
How does the footer look like after the change?

Reply:
It is a bit too much effort for me to run it locally just to check a small design change 😅

Reply:
Added a screenshot. Totally forgot to do that :P